src/style.css

```diff
@@ -1 +1,67 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+.page {
+  color: rgba(61, 78, 97, 1);
+  box-sizing: border-box;
+}
+
+.search-bar {
+  margin-top: 20px;
+}
```

Comment:
Be consistent with your margins. You've used `margin-top` here, so make sure to use `margin-top` consistently throughout your CSS instead of using `margin-bottom` elsewhere.